app/scripts/controllers/transactions/index.js

```diff
@@ -695,6 +673,9 @@ export default class TransactionController extends EventEmitter {
     this.setTxHash(txId, txHash);
 
     this.txStateManager.setTxStatusSubmitted(txId);
+
+    const { gas } = txMeta.txParams;
+    this._trackTxEvent(txMeta, 'Transaction Submitted', { gas });
```

Comment:
lets track this value as `gas_limit: gas` as its confusing by itself and we'll be using gas_limit exclusively sometime in the future

Reply:
sgtm, done